backend/src/mastodon/status.ts

```diff
@@ -165,12 +175,15 @@ export async function toMastodonStatusFromRow(domain: string, db: Database, row:
 		const actorId = new URL(properties.attributedTo)
 		const acct = urlToHandle(actorId)
 		const author = await actors.getAndCache(actorId, db)
-		const account = await loadExternalMastodonAccount(acct, author)
+		const mastodonAccount: MastodonAccount | null = await loadExternalMastodonAccount(acct, author)
+		if (mastodonAccount === null) {
+			return null
+		}
 
 		// Restore reblogged status
 		status.reblog = {
 			...status,
-			account,
+			account: mastodonAccount,
```

Comment:
Note: rest property must be the last in the pattern, and must not have a trailing comma

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#rest_property